source/unified-test-format/unified-test-format.md

```diff
@@ -534,6 +534,14 @@ The structure of this object is as follows:
         client.
     - `serverApi`: Optional [serverApi](#serverapi) object.
 
+    <span id="entity_client_autoEncryptOpts"></span>
+
+    - `autoEncryptOpts`: Optional object with the following fields:
+        - `kmsProviders`: The same as in [`clientEncryption`](#entity_clientEncryption).
+        - `keyVaultNamespace`: Optional, a namespace to the key vault collection. Defaults to "keyvault.datakeys".
```

Comment:
```suggestion
        - `keyVaultNamespace`: The same as in [`clientEncryption`](#entity_clientEncryption).
```

Suggest making this required to match `clientEncryptionOpts` in the Unified Format and [AutoEncryptionOpts.keyVaultNamespace](https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.md#mongoclient-changes).